_digest/january-2021/next-on-self-sabotage-time-scarcity.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: Next on Self-Sabotage::&ensp;Time Scarcity
```

Comment:
change this to ```title: 'Next on Self-Sabotage: Time Scarcity'```. apparently that is how you add colons in markdown for front matter :0